Makefile

```
@@ -11,13 +12,13 @@ run-prod:
 	npm run start
 
 docker-run-dev:
```

Map the container port to the host port (in our case the http port ~ 80)

```suggestion
docker-run-dev:
	docker run -p $(HOST_PORT):$(CONTAINER_PORT) --name $(IMAGE_NAME) -v node_modules -v $(PWD):/app $(IMAGE_NAME)
```